InnoClinic.AuthorizationAPI/API/API.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
 	<ItemGroup>
+		<PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="8.0.0" />
```

Comment:
Update EF packages versions to 8.0.x (now the latest X is 14, as I remember), because early versions of packages could have vulnerabilities